package.json

```diff
@@ -11,6 +11,7 @@
     "jsonschema:generate-blocks": "mkdir -p ./dist/resources/validationSchema/current && node scripts/jsonschema-generate-blocks.js",
     "build": "tsc --declaration; yarn jsonschema:generate-container && yarn jsonschema:generate-blocks",
     "clean": "rm -f tsconfig.tsbuildinfo && rm -rf dist/__tests__ && rm -rf dist/domain && rm -rf dist/flow-spec && rm -rf dist/model",
+    "copy-rc4": "cp dist/resources/validationSchema/current/* dist/resources/validationSchema/1.0.0-rc4",
```

Comment:
I'd suggest we follow the existing script keys, what do you think about `jsonschema:update-rc4` instead ?

- [ ] use `jsonschema:` prefix

Reply:
And thanks for adding this, really appreciate 💯